code/game/machinery/recycle_vendor.dm

```diff
@@ -575,6 +620,7 @@
 	materials_stored[materialtosell] -= amount
 	var/datum/transaction/T = new(cost, my_account.get_name(), "Export", TRADE_SYSTEM_IC_NAME)
 	T.apply_to(my_account)
+	taxdebt += cost/16 // taxed double
```

Comment:
If something is exported, the value of it is taxed for a value of 1/16th of the sale value.